CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
+* Support arithmetric operations in query parser. You can now say something like "(age * 2) + 5 > child.age".
```

Comment:
```suggestion
* Support arithmetic operations (+, -, *, /) in query parser. You can now query with something like "(age * 2) + 5 > child.age".
```

Reply:
Please ensure a full description of all features, so it can be documented in the relevant SDKs. Perhaps just an example that shows all possibilities?

Reply:
I have now enhanced description.